agent/conf/agent.properties

```diff
@@ -266,3 +266,7 @@ iscsi.session.cleanup.enabled=false
 # Automatically clean up iscsi sessions not attached to any VM.
 # Should be enabled for users using managed storage for example solidfire.
 # Should be disabled for users with unmanaged iscsi connections on their hosts
+
+# This parameter specifies the heartbeat update timeout in ms; The default value is 60000ms (1 min).
+# Depending on the use case, this timeout might need increasing/decreasing.
+# heartbeat.update.timeout=60000
```

Comment:
@GutoVeronezi 
please add a "\n" to the end of the file.
it caused failures in trillian test that fresh installed host cannot be added to zone.

in /etc/cloudstack/agent/agent.properties,
```
# heartbeat.update.timeout=60000keystore.passphrase=RFXZya8a4esacWaG
```
should be
```
# heartbeat.update.timeout=60000
keystore.passphrase=RFXZya8a4esacWaG
```

Reply:
@weizhouapache done, thanks!